VS_Project/Pokerapp/Pokerapp/end-game.cshtml

```diff
@@ -0,0 +1,55 @@
+﻿@using WebMatrix.Data
+
+@{
+    string connectionString = @"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\Database.mdf;Integrated Security=True";
+    string provider = "System.Data.SqlClient";
+    Database db = Database.OpenConnectionString(connectionString, provider);
+
+    int roomcode = 1234567;
+
+    if (IsPost && !Request.Form["quit"].IsEmpty())
+    {
+        int endgamequery = (int)db.Execute("UPDATE Players SET Tbl_nr = NULL, Rebuy = NULL, Active = NULL WHERE Room_Code = @0", roomcode);
+
+
+        Response.Redirect("insertpagehere.cshtml");
```

Comment:
Interessante redirect moet ik zeggen. Ik zou hier een comment van maken als ik jou was, anders crasht je programma zodra je hier op quit klikt. Op zich ook wel effectief, maar het is wel irritant als je hem elke keer opnieuw op moet starten.